### PR TITLE
Add speaker infos

### PR DESCRIPTION
**Pull request by netochaves**
Added info about role and company for each speaker #2 

image

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -8,24 +8,44 @@ module.exports = {
     speakersData: [
       {
         name: 'Samuel Flores',
+        role: 'Desenvolvedor Back-end',
+        company:{
```
Comment (willamesoares) [inline gatsby-config.js]: Missing space after colon

File: gatsby-config.js [hunk 2]
```diff
@@ -8,24 +8,44 @@ module.exports = {
...
+          url: 'http://codeminer42.com'
+        },
         talkTitle: 'Introdução à Regex',
         talkDescription: '',
         imageSrc: 'samuel-flores.jpg'
       },
       {
         name: 'Rodrigo Ulisses',
+        role: 'Desenvolvedor Back-end',
+        company:{
```
Comment (willamesoares) [inline gatsby-config.js]: Missing space after colon

File: gatsby-config.js [hunk 3]
```diff
@@ -8,24 +8,44 @@ module.exports = {
...
+          url: '#'
+        },
         talkTitle: 'Precificando projetos',
         talkDescription: '',
         imageSrc: 'rodrigo-ulisses.jpeg'
       },
       {
         name: 'Neto Chaves',
+        role: 'Desenvolvedor Front-end',
+        company:{
```
Comment (willamesoares) [inline gatsby-config.js]: Missing space after colon

File: src/@lekoarts/gatsby-theme-cara/components/SpeakerCard.tsx [hunk 4]
```diff
@@ -7,9 +7,11 @@ type SpeakerCardProps = {
   children: React.ReactNode
   bg: string
   imageSrc: string
+  role: string
+  company: string
```
Comment (willamesoares) [inline src/@lekoarts/gatsby-theme-cara/components/SpeakerCard.tsx]: Is `company` a string though?

File: src/@lekoarts/gatsby-theme-cara/components/SpeakerCard.tsx [hunk 5]
```diff
@@ -47,6 +49,24 @@ const SpeakerCard = ({ name, children, bg, imageSrc }: SpeakerCardProps) => (
     >
       {name}
     </div>
+    <div 
+      sx={{
+        letterSpacing: `wide`,
+        fontSize: [1, 2],
+        lineHeight: 1,
+      }}>
```
Comment (willamesoares) [inline src/@lekoarts/gatsby-theme-cara/components/SpeakerCard.tsx]: Keep  `>` in the next line
```suggestion
      }}
    >
```

**Comment by netochaves** [comment]
I think we need a lint or prettier :smile:

**Comment by willamesoares** [comment]
hey @netochaves  I noticed a few things

`Palestrantes` is showing up at the bottom when we load the website on mobile
Screen Shot 2019-09-10 at 12 30 53 AM

`Sobre` is over the last speaker card on mobile
Screen Shot 2019-09-10 at 12 31 00 AM

The speaker position could have a smaller font size and the title talk can have less space between lines
Screen Shot 2019-09-10 at 12 32 12 AM

Can you tackle those issues so we can merge this?

**Comment by netochaves** [comment]
> hey @netochaves I noticed a few things
> 
> `Palestrantes` is showing up at the bottom when we load the website on mobile
> Screen Shot 2019-09-10 at 12 30 53 AM
> 
> `Sobre` is over the last speaker card on mobile
> Screen Shot 2019-09-10 at 12 31 00 AM
> 
I think we should wait #7 and [#1 of theme](https://github.com/lntrodev/gatsby-theme-cara/pull/1) to be merged because I changed the sections spacing on this prs, them we rebase and see if we need any change.
> The speaker position could have a smaller font size and the title talk can have less space between lines
> Screen Shot 2019-09-10 at 12 32 12 AM

Done on e54033f809be1af77643a08f6bc0d58abedb0f67

**Pull request merged**